client.go

```diff
@@ -42,6 +43,14 @@ type Client interface {
 	// offset, OffsetNewest for the offset of the message that will be produced next, or a time.
 	GetOffset(topic string, partitionID int32, time int64) (int64, error)
 
+	// Coordinator returns the coordinating broker for a consumer group. It will return a locally cached
+	// value if it's available. This may be stale, in which case you should call RefreshCoordinator
+	// to update the locally cached value.
+	Coordinator(consumerGroup string) (*Broker, error)
+
+	// RefreshCoordinator refreshes the coordinator for a consumer group.
```

Comment:
More specifically: we *don't* want anybody to fetch from the cluster without updating the cache. What good would that do? Splitting the two methods gives us that flexibility, yes, but I think that's flexibility we should be avoiding. The cache is the source of truth. There is a single black box method that updates the cache.